### DISCUSSION TITLE
How do you give yourself +5000 claim blocks shovel?

### DISCUSSION THREAD

**Discussion by ToxicWaterOP** [GriefPrevention/GriefPrevention#2016] [Q&A]
So, i've seen a server that has a shovel saying "+5000 Claim Blocks", and I wanted to know how I could get this on my server? 
![Screenshot 2023-03-09 145824](https://user-images.githubusercontent.com/97135702/224179495-a9cfe751-f432-4d49-9007-fd447ec11182.png)

**Comment by Jikoo** [comment]
I assume it runs a command to give you claim blocks rather than give you an actual shovel.
